kf_lib_data_ingest/common/concept_schema.py

```diff
@@ -138,6 +138,8 @@ class EVENT_AGE(QuantityMixin):
         CONCENTRATION_MG_PER_ML = None
         VOLUME_UL = None
         SAMPLE_PROCUREMENT = None
+        DBGAP_CONSENT_CODE = None
```

Comment:
Is there anything dbgap-specific about the format of these codes?

Reply:
related to your other comment

> > for non-dbgap studies or for studies where dbgap has not released the data
> 
> Where is dbgap data coming from if not from dbgap? If it's coming from somewhere else then it sounds like it isn't dbgap anything.

you're correct. this could be any type of consent code. perhaps it was a mistake to name it `DBGAP_CONSENT_CODE` - referencing the target api. So a better name would be?

```suggestion
        CONSENT_CODE = None
```

Reply:
I don't know. I think that depends on why the dataservice calls its field "dbgap".

Reply:
New question. Is there only one code per specimen?

Reply:
yes - only one code per specimen

Reply:
As far as I remember, the assignment of specimen-level consent code is dbGaP-specific (I am not sure about whether it can have multiple values).

Reply:
I am actually down for `CONSENT_CODE` since it is inclusive of other scenarios. Why don't we double-check with Allison and Bailey?